update-outdoor_temp.ts

```diff
@@ -0,0 +1,133 @@
+import axios from "axios";
```

Comment:
Can you rename this file to `update-outdoor-temp.ts` for consistency?

Reply:
Done!